app/frontend/src/components/admin/ServiceClients.vue

```diff
@@ -0,0 +1,231 @@
+            </v-tooltip>
+          </div>
+        </template>
+        <template v-slot:item.test="{item}">
+          <div v-if="item.test">
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-icon color="green" v-on="on">check</v-icon>
+              </template>
+              <span>updated: {{ formatEnvDate(item.environments.TEST) }}</span>
+            </v-tooltip>
+          </div>
+        </template>
+        <template v-slot:item.prod="{item}">
+          <div v-if="item.prod">
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-icon color="green" v-on="on">check</v-icon>
+              </template>
+              <span>Updated: {{ formatEnvDate(item.environments.PROD) }}</span>
+            </v-tooltip>
+          </div>
+        </template>
+
+        <!-- expanded row -->
+        <template v-slot:expanded-item="{ headers, item }">
+          <td :class="[responsiveCell]"></td>
+          <td :colspan="headers.length - 1" :class="[responsiveCell]">
+            <div class="kc-expanded">
+              <!-- app details -->
+              <strong>Name: </strong>
+              <span>{{ item.name }}</span>
+            </div>
+
+            <div class="kc-expanded">
+              <!-- app details -->
+              <strong>Created: </strong>
+              <span>{{ formatDate(item.acronymDetails.createdAt) }}</span>
+            </div>
+
+            <div class="kc-expanded">
+              <!-- users -->
+              <strong>Users: </strong>
+              <ul>
+                <li
+                  v-for="(user, index) in item.users"
+                  :key="index"
+                >{{ user.user.firstName + ' ' + user.user.lastName + ((item.users.length > (index + 1)) ? ',' : '') }}</li>
+              </ul>
+            </div>
+          </td>
+        </template>
+      </v-data-table>
+    </v-container>
+  </v-card>
+</template>
+
+<script>
+import keycloakService from '@/services/keycloakService';
+
+export default {
+  name: 'Clients',
+  computed: {
+    responsiveCell () {
+      return (this.$vuetify.breakpoint.name == 'xs') ? 'v-data-table__mobile-table-row' : '';
+    }
+  },
+  data() {
+    return {
+      // vuetify data table
+      search: '',
+      headers: [
+        { text: '', value: 'data-table-expand' },
+        { text: 'Application', align: 'start', value: 'acronym' },
+        { text: 'DEV', value: 'dev' },
+        { text: 'TEST', value: 'test' },
+        { text: 'PROD', value: 'prod' }
+      ],
+      serviceClients: [],
+      loading: true,
+      showAlert: false,
+      alertType: null,
+      alertMessage: '',
+      expanded: []
+    };
+  },
+  methods: {
+
+    formatEnvDate(env) {
+      return (env && env.created) ?  this.formatDate(env.created): '-';
+    },
+    formatDate(date){
+      return new Date(date).toLocaleString();
+    },
```

Comment:
These two functions can probably be chained together into one - this is likely why the tooltip and created date outputs are inconsistent as you're calling them both differently.